### PR TITLE
Task material fix editable list

### PR DESCRIPTION
**Pull request by ddufresne**
https://isidor.numigi.net/web#id=11460&view_type=form&model=project.task&action=292&active_id=206&menu_id=200

### CODE REVIEW

File: project_wip_material/static/src/fix_editable_list.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    environment.
+*/
+odoo.define("project_wip_material.fix_editable_list", function(require) {
+"use strict";
+
+var dom = require("web.dom");
+var oldGetSelectionRange = dom.getSelectionRange
+
+dom.getSelectionRange = function(node){
+    return node === undefined ? {start: 0, end: 0} : oldGetSelectionRange(node)
```
Comment (ddufresne) [inline project_wip_material/static/src/fix_editable_list.js]: Codacy Issue found: [Unexpected use of undefined. (no-undefined)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3552300)

File: project_wip_material/static/src/fix_editable_list.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    This bug could be fixed in the core of Odoo.
+    However, Odoo does not prioritize bugs which can not be reproduced in a vanilla
+    environment.
+*/
+odoo.define("project_wip_material.fix_editable_list", function(require) {
+"use strict";
+
+var dom = require("web.dom");
+var oldGetSelectionRange = dom.getSelectionRange
```
Comment (ddufresne) [inline project_wip_material/static/src/fix_editable_list.js]: Codacy Issue found: [Missing semicolon. (semi)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3552300)

File: project_wip_material/static/src/fix_editable_list.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+dom.getSelectionRange = function(node){
+    return node === undefined ? {start: 0, end: 0} : oldGetSelectionRange(node)
+}
```
Comment (ddufresne) [inline project_wip_material/static/src/fix_editable_list.js]: Codacy Issue found: [Missing semicolon. (semi)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3552300)

File: project_wip_material/static/src/fix_editable_list.js (hunk 1 above)
Comment (ddufresne) [inline project_wip_material/static/src/fix_editable_list.js]: Codacy Issue found: [Missing semicolon. (semi)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3552300)

File: project_wip_material/static/src/fix_editable_list.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    web/static/src/js/views/list/list_editable_renderer.js
+
+    Because the next field (consumed_qty) is readonly, there is no focused element.
+    Therefore, the variable focusedElement is undefined.
+
+    This bug could be fixed in the core of Odoo.
+    However, Odoo does not prioritize bugs which can not be reproduced in a vanilla
+    environment.
+*/
+odoo.define("project_wip_material.fix_editable_list", function(require) {
```
Comment (ddufresne) [inline project_wip_material/static/src/fix_editable_list.js]: Codacy Issue found: ['odoo' is not defined. (no-undef)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3552300)

File: project_wip_material/static/src/fix_editable_list.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+"use strict";
+
+var dom = require("web.dom");
+var oldGetSelectionRange = dom.getSelectionRange;
+
+dom.getSelectionRange = function(node){
+    return node === undefined ? {start: 0, end: 0} : oldGetSelectionRange(node);
```
Comment (ddufresne) [inline project_wip_material/static/src/fix_editable_list.js]: Codacy Issue found: [Unexpected use of undefined. (no-undefined)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=3552300)

**Review by foutoucour — APPROVED** [review]

**Pull request merged**